package.bat

```diff
@@ -0,0 +1,13 @@
+echo Please ensure PYTHON is set to the directory of Python installation at version 3.6.
+set OLD_PATH=%PATH%
+PATH=%PYTHON%;%PYTHON%\Scripts;%PATH%
+python --version
+pip --version
+pip install cx_Freeze
```

Comment:
Alright; I thought `requirements.txt` should contain dependencies for all "actions" for a project, but splitting building and running definitely makes sense.

It looks OK, I just want to run `setup.py` before merging and I'm having problems with installation of `cx_Freeze` (most probably because of my `gcc` version).